Src/CSharpier.Tests/FormattingTests/BaseTest.cs

```diff
@@ -13,16 +10,14 @@ namespace CSharpier.Tests.FormattingTests;
 
 public class BaseTest
 {
+#pragma warning disable CS8618
     private DirectoryInfo rootDirectory;
+#pragma warning restore CS8618
 
     [OneTimeSetUp]
     public void Setup()
     {
-        this.rootDirectory = new DirectoryInfo(Directory.GetCurrentDirectory());
-        while (this.rootDirectory.Name != "CSharpier.Tests")
-        {
-            this.rootDirectory = this.rootDirectory.Parent;
-        }
+        this.rootDirectory = DirectoryFinder.FindParent("CSharpier.Tests");
```

Comment:
Since this is a static method, why not just call it in the field initializer?
That way, there's no need to suppress `CS8618`.